test/validation/api/pktio/pktio.c

```diff
@@ -3285,7 +3285,7 @@ static void pktio_test_pktout_compl(bool use_plain_queue)
 			} else {
 				CU_ASSERT_FATAL(pktio_capa.tx_compl.queue_type_sched != 0);
 			}
-
+			CU_ASSERT_FATAL(pktio_capa.config.pktout.bit.tx_compl_ena == 1);
```

Comment:
Actually, we should remove .config field from pktio capa, as it makes automatically everything optional.

It should be enough for application to check only pktio_capa.tx_compl.mode_all. So, this check should not be added.

Reply:
OK, currently, pktio implementation specific capability is signaled with the bit (there's a hefty amount of other features as well using the `config` fields for capa). Anyways, I can switch to use the mode field in pktios.